app/modules/kit-library/assets/js/components/kit-dialog.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+import KitAlreadyAppliedDialog from './kit-already-applied-dialog';
+import ApplyKitDialog from './apply-kit-dialog';
+
+export default function KitDialog( props ) {
+	const [ applyAnyway, setApplyAnyway ] = useState( false );
+
+	if ( elementorAppConfig[ 'import-export' ].importSessions.length && ! applyAnyway ) {
```

Comment:
Maybe for better readability.

```suggestion
const kitAlreadyApplied = elementorAppConfig[ 'import-export' ].importSessions.length;
	if (  kitAlreadyApplied && ! applyAnyway ) {
```